runtime/gocont.go

```diff
@@ -90,8 +90,16 @@ func (c *GoCont) RunInThread(t *Thread) (next Cont, err *Error) {
 		return nil, err
 	}
 	t.RequireCPU(1) // TODO: an appropriate amount
+
+	t.goFunctionCallDepth++
+	defer func() { t.goFunctionCallDepth-- }()
+
+	if t.goFunctionCallDepth > maxGoFunctionCallDepth {
+		return nil, NewErrorS("Go stack overflow")
```

Comment:
I'd avoid exposing the host language to the (Lua) user. `Maximum stack depth is reached` might be a good alternative generic message.

Reply:
Yes good point - I want to use the words "stack overflow" because that is the terminology C Lua uses.  I could just call it "stack overflow" I guess?

Reply:
Done!